pkg/service/replay/replay.go

```diff
@@ -380,13 +381,19 @@ func (r *replayer) RunTestSet(ctx context.Context, testSetID string, testRunID s
 			utils.LogError(r.logger, err, "failed to simulate request")
 			break
 		}
+		consumedFilteredMocks, err := r.instrumentation.GetConsumedFilteredMocks(runTestSetCtx, appID)
```

Comment:
We need to print the unfiltered mocks as well because the dependencies like postgreSQL, generic have only config mocks `unfiltered` mocks. So need to merge both the consumed filtered and unfiltered mocks in sorted fashion.

Reply:
got it but I think we don't need sorting because when we update mocks we are going to read the mock file and  from that array of mocks that are read we will remove few. I guess we don't need sorting.

Reply:
Are suggesting sorting for printing the log..?

Reply:
done the changes you can review now